src/Common/Common.Api/Datasets/IDatasetsClient.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT License.
+*/
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.PowerBI.Common.Api.Datasets
```

Comment:
Consider calling this Data like the module since it includes both datasets and datasources.

Reply:
Yeah we should change, seperate PR